### DISCUSSION TITLE
Let user resize the result window by dragging the right-side bottom corner

### DISCUSSION THREAD

**Discussion by raindropsfromsky** [Flow-Launcher/Flow.Launcher#1172] [Ideas]
Often the search result shows file name or path that is too long for the result window.
It would be very convenient if the user can drag the right-hand bottom corner to resize the window.

Flow Launcher should remember this size for the next search operation.

**Comment by taooceros** [comment]
@onesounds

**Reply by onesounds** [reply]
As you know, it's not easy to implement because of the structure. Long file names and paths need a way to make them easier to see.

**Reply by taooceros** [reply]
We can have a minimum width and height probably? But also the variable height may make the font size display weird.

**Reply by taooceros** [reply]
I agree with you though. We can think of an alternative besides simply allowing stretching.

**Reply by raindropsfromsky** [reply]
Just to clarify, I have set the width of the search box itself to its maximum permissible width.
IMHO the tooltip should be maximized automatically but not allowed to spill over the search box.
In other words, the tooltip's shape can be from the mouse position to the right edge of the search box.
(Come to think of it, why not even allow it to spill over the FL's search box? No harm in it!)

At present, the tooltip remains unnaturally narrow, not even utilizing the footprint of FL's search box.

BTW the FL's search box has the maximum defined width of 900 pixel, while my monitor is FHD.
That's another level of inefficiency: Why not let it maximize itself to full screen mode (both width and height)?

**Reply by jjw24** [reply]
> why not even allow it to spill over the FL's search box?

I am using 1.9.4 with default width and the tooltip is spilling over the search box, as in it extends beyond the result box.

>TW the FL's search box has the maximum defined width of 900 pixel, while my monitor is FHD.
That's another level of inefficiency: Why not let it maximize itself to full screen mode (both width and height)?

Not sure about full screen, but I think we can increase the 900 pixel width limit.

**Comment by onesounds** [comment]
- I misunderstood this request. It is possible to increase the maximum width size.
- I increase the maximum width size and modified little layouts. https://github.com/Flow-Launcher/Flow.Launcher/pull/1355
